lib/model.js

```diff
@@ -1412,6 +1412,13 @@ module.exports = (function() {
    * @return {Promise<undefined>}
    */
   Model.prototype.destroy = function(options) {
+    var self = this
+      , instances;
+
+    if (!options || !(options.where || options.truncate)) {
```

Comment:
`Model.destroy({ truncate: false })` shouldn't be allowed anyway? It's the same as `Model.destroy()`, which isn't allowed.